### PR TITLE
Added Source URL. This fixes #1234 issue.

### PR DESCRIPTION
**Pull request by sagarhani**
This will fix #1234 issue.

### CODE REVIEW

**Comment by MariagraziaAlastra** [comment]
Hi @sagarhani, looks like in this PR you included all the changes from [the Gmail CheatSheet PR](https://github.com/duckduckgo/zeroclickinfo-goodies/pull/1232). 
Would you mind fixing that? Thanks!

**Comment by sagarhani** [comment]
@MariagraziaAlastra  Okay. I got it. I'll fix it. Thanks!

**Comment by jagtalon** [comment]
@sagarhani any updates? :)

**Comment by sagarhani** [comment]
@jagtalon The [gmail cheat sheet](https://github.com/duckduckgo/zeroclickinfo-goodies/pull/1232) has been fixed along with this PR.

File: lib/DDG/Goodie/RouterPasswords.pm [hunk 1]
```diff
@@ -1,4 +1,4 @@
-package DDG::Goodie::RouterPasswords;
+    package DDG::Goodie::RouterPasswords;
```
Comment (moollaza) [inline lib/DDG/Goodie/RouterPasswords.pm]: This isn't necessary

File: lib/DDG/Goodie/WhereAmI.pm [hunk 2]
```diff
@@ -1,35 +0,0 @@
-package DDG::Goodie::WhereAmI;
```
Comment (moollaza) [inline lib/DDG/Goodie/WhereAmI.pm]: Is this supposed to be in the PR?

File: share/goodie/cheat_sheets/gmail.json [hunk 3]
```diff
@@ -0,0 +1,117 @@
...
+        }, {
+            "key": "[Shift] + [a]",
+            "val": "Reply to all message recipients"
+            
+        }, {
+            "key": "f",
+            "val": "Forward a message"
+            
+        }, {
+            "key": "[Shift] + [f]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/gmail.json]: Please remove the `+`, as the cheat sheet syntax mentions, they shouldn't be used any longer :smile:

File: share/goodie/cheat_sheets/gmail.json [hunk 4]
```diff
@@ -0,0 +1,117 @@
...
+            "val": "Forward a message in a new window"
+            
+        }, {
+            "key": "Esc",
+            "val": "Escape from input field"
+            
+            }],
+        "Navigate": [{
+            "key": "[Tab] then [Enter]",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/gmail.json]: Please replace `then`, with `,`

**Comment by sagarhani** [comment]
Oops! I guess this is by mistake and this has lot of issues!! 
Actually I have corrected all these syntax in #1232 . Please check #1232 .
Better I close this PR and send a new PR to fix #1234

**Pull request closed without merging**